doc/tutorial/Tutorial.tex

```diff
@@ -1562,14 +1557,13 @@ \section{Managing the context}
 SearchHead le.
 \end{coq_example}
 
-A new and more convenient search tool is \verb:SearchPattern:
-developed by Yves Bertot. It allows finding the theorems with a
-conclusion matching a given pattern, where \verb:_: can be used in
+The \verb:Search: commands also allows finding the theorems
+containing a given pattern, where \verb:_: can be used in
 place of an arbitrary term. We remark in this example, that \Coq{}
```

Comment:
"We remark in this example, that" sounds awkward to me.

How about: "Note that in this example, " or simply "In this example, "